Documentation/security/threat-model.rst

```diff
@@ -725,21 +687,23 @@ Overall Recommendations
 To summarize the recommended controls to be used when configuring a
 production Kubernetes cluster with Cilium:
 
-1. Ensure that Kubernetes roles are scoped correctly to the requirements of your
+#. Ensure that Kubernetes roles are scoped correctly to the requirements of your
```

Comment:
Good practice on numbering. Have you verified that your local build renders ordered steps correctly?